CONTRIBUTING.md

```diff
@@ -158,29 +170,86 @@ Please, follow the link to [create an issue](https://github.com/arrow-kt/arrow/i
 
 ### How to create a pull request
 
+The easiest way to contribute to Arrow is to create a branch from a fork, and then create a PR on Github from your branch.
+
+Arrow is a large project that uses several tools to verify that the code is formatted consistently, and that we don't break downstream projects that rely on Arrow's API across versions. 
+
+For code formatting we use [Spotless](https://github.com/diffplug/spotless/tree/main/plugin-gradle) with [KtFmt](https://github.com/facebookincubator/ktfmt) and for API binary compatibility we use [Binary Compatibility Validator](https://github.com/Kotlin/binary-compatibility-validator). They need to run before you commit and push your code to Github.
+
+If you've included those changes for binary compatibility and formatted the code correctly it's time to open your PR and get your contribution into Arrow. Thanks ahead of time for your effort and contributions 🙏
```

Comment:
> hanks ahead of time for your effort and contributions 🙏
❤️

Reply:
This part was actually originally your suggestion ;) I think it's very good too!